packages/react-native-win32/.npmignore

```diff
@@ -6,7 +6,7 @@
 /lib/tester/index.win32.bundle*
 /lib/tester/index.win32.dev.bundle*
 /scripts
-/src
+/src/**/*.js
```

Comment:
I think this might also include some stuff like tester assets (PNGs) that might not be wanted